### PR TITLE
Geotagging plugin

### PR DESCRIPTION
**Pull request by bkueng**
This adds a camera plugin that writes images with a fixed interval and geotags.

needs the `exiftool` command to be installed.
This PR includes #48

### CODE REVIEW

File: src/gazebo_geotagged_images_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  destHeight_ = height_;
+  if (sdf->HasElement("height")) {
+	destHeight_ = sdf->GetElement("height")->Get<int>();
+  }
+
+
+  //check if exiftool exists
+  if (system("exiftool -ver &>/dev/null") != 0) {
+    gzerr << "exiftool not found. geotagging_images plugin will be disabled" << endl;
+    gzerr << "On Ubuntu, use 'sudo apt-get install libimage-exiftool-perl' to install" << endl;
```
Comment (julianoes) [inline src/gazebo_geotagged_images_plugin.cpp:113]: This should probably be added to the README.md.

File: src/gazebo_geotagged_images_plugin.cpp [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  this->depth_ = this->camera_->ImageDepth();
+  this->format_ = this->camera_->ImageFormat();
+#else
+  this->width_ = this->camera_->GetImageWidth();
+  this->height_ = this->camera_->GetImageHeight();
+  this->depth_ = this->camera_->GetImageDepth();
+  this->format_ = this->camera_->GetImageFormat();
+#endif
+
+  if (sdf->HasElement("robotNamespace"))
```
Comment (julianoes) [inline src/gazebo_geotagged_images_plugin.cpp]: Nitpick: brackets please.

File: src/gazebo_gst_camera_plugin.cpp [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+  gstTimestamp += GST_BUFFER_DURATION(frameBuffer);
+
+  GstFlowReturn ret;
+  g_signal_emit_by_name(appsrc, "push-buffer", frameBuffer, &ret);
+
+  frameBufferMutex.unlock();
+
+  if (ret != GST_FLOW_OK) {
+    /* something wrong, stop pushing */
+	gzerr << "g_signal_emit_by_name failed" << endl;
```
Comment (julianoes) [inline src/gazebo_gst_camera_plugin.cpp]: indent

**Review by julianoes — COMMENTED** [review]
Reviewed, made some small comments, found nothing major.

**Comment by LorenzMeier** [comment]
@bkueng @julianoes Good to go in?

**Comment by bkueng** [comment]
rebased & applied

**Pull request closed without merging**